kiwi/bootloader/config/base.py

```diff
@@ -298,12 +298,15 @@ def get_boot_path(self, target='disk'):
                 filesystem = self.xml_state.build_type.get_filesystem()
                 volumes = self.xml_state.get_volumes()
                 if filesystem == 'btrfs' and volumes:
-                    # boot or boot/grub2 on a subvolume prevents grub from
-                    # finding its config file
+                    # grub boot data paths must not be in a subvolume
+                    # otherwise grub won't be able to find its config file
+                    grub_boot_data_paths = ['boot', 'boot/grub', 'boot/grub2']
```

Comment:
The order in this case is immaterial as we are just looking that a subvolume in the btrfs case is not pointing to any possible grub2 boot path. So this is basically to check for a limitation of grub2's btrfs support and not for the grub2 installation in the system

I agree to the variable naming and will adapt it accordingly